src/WebJobs.Script/Workers/Rpc/FunctionRpcService.cs

```diff
@@ -81,6 +81,10 @@ public override async Task EventStream(IAsyncStreamReader<StreamingMessage> requ
                     while (await messageAvailable());
                 }
             }
+            catch (Exception rpcException)
+            {
+                _logger.LogError(rpcException, "Exception encountered while listening to EventStream");
```

Comment:
log and throw. That would keep the existing behavior but we will have more logs to analyze.

Reply:
Thanks for the heads up, silly me I thought the exception was not getting thrown for some reason. Fixed!